packages/flutter/lib/src/material/dropdown_menu.dart

```diff
@@ -135,6 +138,8 @@ class DropdownMenu<T> extends StatefulWidget {
     this.controller,
     this.initialSelection,
     this.onSelected,
+    this.focusNode,
+    this.showVirtualKeyboard = true,
```

Comment:
Giving the keyboard focus to a Dropdown menu on mobile doesn't seem like the common case, so maybe we should make this a nullable property and compute a default value (if needed) based on the target platform. This would be an incompatible change however DropdownMenu hasn't been out for long.

Reply:
Fixed! On mobile platforms, the text field will be focusable only when the `showVirtualKeyboard` is true now. Thanks so much for the suggestion. Please let me know if there're any problems:)